tests/e2e/specs/merchant/dispute-submit-losing.spec.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { merchant, shopper, uiUnblocked } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+import { merchantWCP, uiLoaded } from '../../utils';
+import { fillCardDetails, setupProductCheckout } from '../../utils/payments';
+
+let orderId;
+
+describe( 'Disputes > Submit losing dispute', () => {
+	beforeAll( async () => {
+		await page.goto( config.get( 'url' ), { waitUntil: 'networkidle0' } );
+
+		// Place an order to dispute later
+		await setupProductCheckout(
+			config.get( 'addresses.customer.billing' )
+		);
+		const card = config.get( 'cards.disputed-unreceived' );
+		await fillCardDetails( page, card );
+		await shopper.placeOrder();
+		await expect( page ).toMatch( 'Order received' );
+
+		// Get the order ID so we can verify it in the disputes listing
+		const orderIdField = await page.$(
+			'.woocommerce-order-overview__order.order > strong'
+		);
+		orderId = await orderIdField.evaluate( ( el ) => el.innerText );
+
+		await merchant.login();
+	} );
+
+	afterAll( async () => {
+		await merchant.logout();
+	} );
+
+	it( 'should process a losing dispute', async () => {
+		await merchantWCP.openDisputes();
+
+		await uiLoaded();
+
+		// Verify a dispute is present with a proper ID and open it
+		await page.waitForSelector( '.woocommerce-table__item > a', {
+			text: orderId,
+		} );
+		await expect( page ).toMatchElement( '.woocommerce-table__item > a', {
+			text: orderId,
+		} );
+		await expect( page ).toClick( '.woocommerce-table__clickable-cell' );
+
```

Comment:
I think we may want to add in an `await uiLoaded();` here before we run the expects below on the texts, looking into the tests, it seems like it may not have fully loaded yet by the time we run these. Alternatively, we could replace it with something like:

```
await page.waitForSelector( '.components-card__header', {
	visible: true,
} );
```

Reply:
I tried and it didn't help, but it forced me to think if the problem is with entering the single dispute screen, the previous step of clicking the dispute `.woocommerce-table__clickable-cell`. Will try some workarounds, thanks. I will keep in the code your proposed change because its nice to have to avoid flakiness.

Reply:
Sounds good! One thing I ended up having to do for refunds to avoid some of that (especially as the link to the transaction details opened in a new tab) was pull out the transaction ID from the anchor tag. Not sure if this is will work here, but it may streamline some of the clicking around being done to avoid flakiness.

For example, we could do similar logic like what I have in the refund flow:

```
		const paymentDetailsLink = await page.$eval(
			'p.order_number > a',
			( anchor ) => anchor.getAttribute( 'href' )
		);
		await Promise.all( [
			page.goto( paymentDetailsLink, {
				waitUntil: 'networkidle0',
			} ),
			uiLoaded(),
		] );

		await uiLoaded();
```

Then we can find the line item (`li.woocommerce-timeline-item`) that is a link to `View dispute`, as in this screenshot:

<img width="1379" alt="Screen Shot 2021-05-13 at 1 50 27 PM" src="https://user-images.githubusercontent.com/71906536/118179325-62719300-b3f2-11eb-88a9-08b03f4c115c.png">

Anyway just a thought as another way to reach the specific dispute for the order in case we're seeing some flakiness from trying to select the order on the `Payments > Disputes` page.

Reply:
Hey, yes I was also thinking about that way... but was unsure. I'll try to think and make something, maybe even some util helper for the future to access disputes. Thanks, I'll consider it.